### PR TITLE
Add position_3d_accuracy for 3DRMS

### PR DESCRIPTION
**Pull request by m-kuhn**
cc @soester

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e9e70c5bf612a0033ce2a8f5e2532a0374d44d18:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-844986739cc5f3dc54ce00573c51a1a97746863f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-844986739cc5f3dc54ce00573c51a1a97746863f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-844986739cc5f3dc54ce00573c51a1a97746863f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-844986739cc5f3dc54ce00573c51a1a97746863f-x86.apk)

File: src/core/expressioncontextutils.cpp [hunk 1]
```diff
@@ -31,6 +31,7 @@ QgsExpressionContextScope *ExpressionContextUtils::positionScope( const GnssPosi
   const qreal magneticVariation = positionInformation.magneticVariation();
   const qreal horizontalAccuracy = positionInformation.hacc();
   const qreal verticalAccuracy = positionInformation.vacc();
+  const qreal hvAccuracy = positionInformation.hvacc();
```
Comment (nirvn) [inline src/core/expressioncontextutils.cpp:34]:
```suggestion
  const qreal horizontalVerticalAccuracy = positionInformation.hvacc();
```

File: src/core/expressioncontextutils.cpp [hunk 2]
```diff
@@ -49,6 +50,7 @@ QgsExpressionContextScope *ExpressionContextUtils::positionScope( const GnssPosi
   scope->addVariable( QgsExpressionContextScope::StaticVariable( QStringLiteral( "position_magnetic_variation" ), magneticVariation, true, true ) );
   scope->addVariable( QgsExpressionContextScope::StaticVariable( QStringLiteral( "position_horizontal_accuracy" ), horizontalAccuracy, true, true ) );
   scope->addVariable( QgsExpressionContextScope::StaticVariable( QStringLiteral( "position_vertical_accuracy" ), verticalAccuracy, true, true ) );
+  scope->addVariable( QgsExpressionContextScope::StaticVariable( QStringLiteral( "position_3d_accuracy" ), hvAccuracy, true, true ) );
```
Comment (nirvn) [inline src/core/expressioncontextutils.cpp:53]:
```suggestion
  scope->addVariable( QgsExpressionContextScope::StaticVariable( QStringLiteral( "position_3d_accuracy" ), horizontalVerticalAccuracy, true, true ) );
```

**Review by nirvn — APPROVED** [review]
Nice -- I'm suggesting a variable name change in expressioncontextutils.cpp to harmonize with other variable names, feel free to apply/ignore.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 406955a29769c309feabf98611098515257eef27:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-ae932c59147eaa7a2c125b3c04415d482370f560-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-ae932c59147eaa7a2c125b3c04415d482370f560-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-ae932c59147eaa7a2c125b3c04415d482370f560-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1590-ae932c59147eaa7a2c125b3c04415d482370f560-x86.apk)

**Pull request merged**